### PR TITLE
UC:65 and UC:53 done for outdoors

### PR DESCRIPTION
**Pull request by Taqi**
User can view outdoor points of interests

### CODE REVIEW

File: src/assets/Languages/language.json [hunk 1]
```diff
@@ -139,6 +142,9 @@
         "depLoy": "Départ de Loyola",
         "depSGW": "Départ de SGW",
         "friday": "Horaire du Vendredi"
-    }
+    },
+    "nearbyPOI": {
+        "title": "Points d'Intérêt à Proximité"
```
Comment (JenniferAbouJaoudi) [inline src/assets/Languages/language.json:147]: In french, we can't really say "Points d'Intérêt à Proximité" ..

File: src/app/components/nearby-points-of-interest/nearby-points-of-interest.component.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.outdoorPOI {
```
Comment (JenniferAbouJaoudi) [inline src/app/components/nearby-points-of-interest/nearby-points-of-interest.component.scss:1]: should be aligned with the other classes.

File: src/app/components/directions/directions.component.ts [hunk 3]
```diff
@@ -79,17 +87,27 @@ export class DirectionsComponent{
...
+    this.setDestination(this.address)
   }
 
-  
+  ngOnInit() 
+  {    
+  }  
+
+  //Sets the destination address
+  setDestination(dest: String)
```
Comment (nickode) [inline src/app/components/directions/directions.component.ts:100]: This is fine as long as the storage module's .ready() function has already been executed. Just a heads up if you see bugs related to this function

File: src/app/components/nearby-points-of-interest/nearby-points-of-interest.component.ts [hunk 4]
```diff
@@ -0,0 +1,143 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
```
Comment (nickode) [inline src/app/components/nearby-points-of-interest/nearby-points-of-interest.component.ts:1]: The structure here is great for the component! Very clean, nice job!

**Pull request merged**